### ISSUE TITLE
LCP is very different in lighthouse and timings under the performance tab

### ISSUE THREAD

**Issue by gregtux** [GoogleChrome/lighthouse#11572]
When I run Lighthouse on https://www.tuxedosonline.com the LCP shows to be 8.6 Seconds.  I get a similar number in the lab results section when I run, page speed insight.  In page speed insight the field data shows 3 seconds.  When I go to the performance tab of chrome inspect the LCP is shown at 3793.7 ms, so about 3..7 seconds.  Why is there such a big difference?

Environment Information
* Affected Channels: 
* Lighthouse version:
* Chrome version: 86 
* Node.js version:
* Operating System:windows 10

**Comment by gregtux** [comment]
Hi

Thank you for looking at my issue but your conclusion that my issue is
"Performance Score Is Different #10657" Is not correct.  I am not
experiencing different results from different locations or devices.  I am
experiencing different results in lighthouse and the performance tab of
chrome's inspect tool.

Vrej Grigorian

**Comment by patrickhulce** [comment]
From the very top of that issue...

> tl;dr - follow the advice in the variance docs, **make sure you're actually testing with the same settings**

Which links to the [FAQ](https://github.com/GoogleChrome/lighthouse/#why-is-the-performance-score-so-low-it-looks-fine-to-me). From the provided information this sounds exactly like the very common frequently asked issue that the performance panel investigation was not done with the same throttling that Lighthouse applies.

Loading the listed URL with the right settings seems to confirm this is the case, so #10657 is the correct duplicate.

![image](https://user-images.githubusercontent.com/2301202/96395116-5a69b180-1189-11eb-9f1d-b5c8c7a36c9e.png)
